src/plugins/Migrations/controllers/commands/migrateGuilds.js

```diff
@@ -0,0 +1,129 @@
+      message.channel.send(`${guild[1].name} already has been created`);
+      continue; //Skips to the next guild;
+    }
+
+    /*For things that can be copy-pasted, with no modifications being made to the data,
+        we will use the system below
+      */
+    let keys = [
+      { oldKey: `prefix.${guildId}`, newKey: "prefix" },
+      { oldKey: `${guildId}.newUserRoles`, newKey: "defaultMemberRoles" },
+      { oldKey: `moderation-server.${guildId}`, newKey: "moderationServer" },
+    ];
+
+    const guildObject = migrationsHandler.copyData(keys);
+
+    guildObject.id = guildId;
+
+    const opRoles = [];
+    const modRole = db.get(`moderator.${guildId}`);
+    const adminRole = db.get(`admin.${guildId}`);
+    if (modRole !== null) opRoles.push(modRole);
+    if (adminRole !== null) opRoles.push(adminRole);
+    if (opRoles.length > 0) guildObject.opRoles = opRoles;
+
+    const levels = [];
+    const levelNums = [
+      "one",
+      "five",
+      "ten",
+      "fifteen",
+      "twenty",
+      "twenty-five",
+      "thirty",
+      "thirty-five",
+      "forty",
+    ];
+    for (const levelNum of levelNums) {
+      levels.push({ name: levelNum, role: db.get(`levels.${guildId}.${levelNum}`) });
+    }
+    guildObject.levels = levels;
+
+    const invites = [];
+    const dbInvites = dbh.invite.getInvites(guildId);
+    if (dbInvites) {
+      Object.keys(dbInvites).forEach((inviteName) => {
+        invites.push({ inviteName: inviteName, inviteCode: dbInvites[inviteName] });
+      });
+      if (invites.length > 0) guildObject.invites = invites;
+    }
+
+    const threadInactivityTime = db.get(`thread-inactivity-time.${guildId}`);
+    if (threadInactivityTime) guildObject.threadInactivityTime = threadInactivityTime * 60; //hours to minutes
+
+    const moderationServer = db.get(`moderationServer.${guildId}`);
+    if (moderationServer) guildObject.moderationServer = moderationServer;
+
+    const modCooldown = db.get(`mod-cooldown.${guildId}`);
+    if (modCooldown) guildObject.modCooldown = parseInt(modCooldown);
+
+    const dbGuild = await Guild.create(guildObject);
+
+    await message.channel.send(
+      `Created guild for **${guild[1].name}** with attributes: \`\`\`${Object.keys(
+        dbGuild.dataValues
+      ).map((attribute) => dbGuild.dataValues[attribute])}\`\`\``
+    );
+    await message.channel.send("Starting to migrate the **important channels...**");
+
+    keys = [
+      { oldKey: `welcome.${guildId}`, newKey: "welcome" },
+      { oldKey: `audit.${guildId}`, newKey: "auditLog" },
+      { oldKey: `total.${guildId}`, newKey: "memberCounter" },
+      { oldKey: `threads-category.${guildId}`, newKey: `threadDirectory` },
+      { oldKey: `teams.${guildId}`, newKey: "teamsAndProjects" },
+      { oldKey: `introductions-channel.${guildId}`, newKey: "introductions" },
+      { oldKey: `invite.${guildId}`, newKey: invites },
+    ];
+
+    const channelsObject = migrationsHandler.copyData(keys);
```

Comment:
I think this function is easier to read if we separate it into one to copy guild and another to copy channels.

Reply:
The purpose of the function is not reuse the code in multiple places. I have now changed `keys` to `guildKeys` and `channelKeys` to make it easier to read.